src/runtime/hexagon_remote/sim_remote.cpp

```diff
@@ -232,21 +232,64 @@ typedef int (*set_runtime_t)(halide_malloc_t user_malloc,
                              void *(*)(const char *),
                              void *(*)(void *, const char *));
 
-int initialize_kernels(const unsigned char *code, int codeLen,
-                       handle_t *module_ptr) {
-    elf_t *lib = obj_dlopen_mem(code, codeLen);
-    if (!lib) {
-        halide_print(NULL, "dlopen_mem failed\n");
-        return -1;
+// This function should be deleted once the hexagon tools impliment the dlopenbuf api.
```

Comment:
How about making it not weak, so we know it (because the build will break) when it appears?